hack/make/.build-deb/control

```diff
@@ -6,7 +6,7 @@ Vcs-Git: git://github.com/docker/docker.git
 
 Package: docker-engine
 Architecture: linux-any
-Depends: iptables, ${misc:Depends}, ${perl:Depends}, ${shlibs:Depends}
+Depends: iptables, debhelper, dh-apparmor, ${misc:Depends}, ${perl:Depends}, ${shlibs:Depends}
```

Comment:
If you use dh-apparmor, you need to Depends the package. It's not a requirement for apparmor itself. A bug in trusty (and possible other distros) has the debhelper requirement missing in the Depends for dh-apparmor.